integration_tests/steps/integration_tests.py

```diff
@@ -220,6 +220,9 @@ def one_sided_market(context, market_type):
     elif market_type == "two-sided-pay-as-clear":
         ConstSettings.IAASettings.MARKET_TYPE = 2
         ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE = BidOfferMatchAlgoEnum.PAY_AS_CLEAR.value
+    elif market_type == "two-sided-external":
+        ConstSettings.IAASettings.MARKET_TYPE = 2
+        ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE = BidOfferMatchAlgoEnum.EXTERNAL.value
```

Comment:
I'm wondering: are these constants (`ConstSettings.IAASettings.xxx`) restored to their default at the end of each integration test? If not, I think we need a teardown process (maybe using [behave's fixtures](https://behave.readthedocs.io/en/stable/fixtures.html))

Reply:
AFAIR each integration test will either set it in the steps or in the setup file that will be run (I suppose all setups set the market type)

Reply:
Hmm...either way we wouldn't be testing the default values. If that doesn't take too long, I would still suggest you to check how fixtures could be used for this: we had the same problem with unit-tests a while ago and I remember that Spyros had to work through a pretty tedious process to fix them.

Reply:
@fievelk this is already implemented `integration_tests/environment.py:59`, it looks all const settings are reversed to default after each scenario